src/rez/system.py

```diff
@@ -88,7 +88,7 @@ def shell(self):
                     args = ['ps', '-o', 'args=', '-p', str(parent_pid)]
                     proc = sp.Popen(args, stdout=sp.PIPE)
                     output = proc.communicate()[0]
-                    shell = os.path.basename(output.strip().split()[0]).replace('-', '')
+                    shell = os.path.basename(output.decode().strip().split()[0]).replace('-', '')
```

Comment:
`output` is `bytes` in python3, so need to call `decode()`

Reply:
We use `text=True`, so it should be string isn't it?

Reply:
We didn't use `text=True` here.   I could pass `text=True ` to `Popen`.

Reply:
See https://github.com/AcademySoftwareFoundation/rez/blob/main/src/rez/system.py#L89.